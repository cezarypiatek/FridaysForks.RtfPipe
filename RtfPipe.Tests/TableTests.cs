using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace RtfPipe.Tests;

[TestClass]
public class TableTests
{
  [TestMethod]
  public void marge_scenario_1() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_2() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_3() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_4() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_5() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_6() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_7() => this.VerifyCurrentScenario();
}
