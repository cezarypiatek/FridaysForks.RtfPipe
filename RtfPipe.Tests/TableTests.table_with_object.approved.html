<div style="font-size:12pt;font-family:&quot;Times New Roman&quot;, serif;">
    <table style="line-height:1.3;font-family:Aptos, sans-serif;font-size:11pt;border-spacing:0;box-sizing:border-box;margin:0 0 0 0.3px;">
        <colgroup>
            <col style="width:295.2px;" />
            <col style="width:295.2px;" />
        </colgroup>
        <tr style="text-align:left;">
            <td style="vertical-align:top;border:1px solid;padding:0 7.2px 10.7px 7.2px;">
                <p style="margin:0 0 10.7px 0;">Sample1
                </p>
                <p style="margin:0 0 10.7px 0;">
                    <br />
                </p>
            </td>
            <td style="vertical-align:top;border:1px solid;padding:0 7.2px 10.7px 7.2px;">
                <img width="96" height="81" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGAAAABRCAYAAAAtvqMwAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAPBSURBVHhe7Zqxi9RAGMW39E+wtbE5LW3SieAfYHkiWEhaG0EOtLIVSW0pnHiF7V6loKB4IBYqFi4IemBzoBwiwlnEvNnvy34zO7PJTvZYOd4PnptJZr5M3pvMLocjQua4dnWz3rp9K1vnz23UUorkABOPjo6yxBBWwNAA9vbeMIQhDA0AnwxhAKsIAGIImawqAIghZLDKACCG0MGpM5fr05vPsrT9fN8ze+fJY2d2SnJLYokZu4xsAIvEABLETF1GMbNjYgAJ1Midl99rcP3B+9bc/YM/7tzZGy9c+8q9d66NvtonZnZMDCCBGnn/6RdnLj7RvnDzlWsDDeXuo8+urX2gmNkxMYAEaqSu7teffnpmg4e739w5fUvQV8fFzI6JASRQI7HNgMPff1179+2BawNsRTj38esv19YtCYK5F7d/JMUAOlAjIbvnAxiuQeiWpGGo1OAuMYAE1kw1G1sOwF6vW5Few6cdA3M37nxIigF0YM3UL2Ll0tZeu/KxNQH7BQypwV1iAAmsmfi1o9itRrcmYH+mQjDXtkMxgA6sWfanp/7ygXRLAuhjx6jBXWIACayZkG41dqXrm6G/kKxiZsfEABKEhi6rmNkxMYAEMVOXUczsmBhAglX+OXqRGEBPYFTMwKFiAD1hAGuGAawZBrBmGMCaYQBrZuh/zk0JdeUWhBBCCCEduD/Ck2zExnykDslEbMxH6pBMxMZ8pI7PuEThelRU9UROxRnX5aioK9fJHh8DOqdGRa+bHPN8BHg4CKljwMRHdTmW5kJSAQx9+HA82mXzrx73qT10Dv0QG/OROoZlJm77po5zCMZj9ZsVMamKHm/B0Dn0Q2zMR+q0YPXjNDR9yPBBUkbrMT5nNVrjJlVdtOexmid1VZjtBCa7LW9+/JzhQSAt5h5FVZm5+dcgf/h0LrNrqbnN0/QfhtQxWFPBonafYxC0mweahYsw9FPx+/cLAGNmxmLMyJuPMd2FodcCs1tSc/MRG/OROgb/4Re3+xw3BKvPSd2AmU3b99Mf3ysA3MNbpaaGu79v4riUe0autUTn5tNcH4bUMQTmLWz3OW5Y+iGD8YHh0e+AkxvA9BVtJ+EmFTM6dQzQXuY1D8Z7Jtlr4fFsnt1bkNb777egBlkJTmVprtu+/jissGn/1pHmoeWcO195D+4MMyt4bryZQ2tkOFfTZ/GXsDnvQB29hvp+KOHcLE3/YUgdkonYmI/UIZmIjflIHZKJ2EgIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCEnmtHoHxdxzIZ6u7rGAAAAAElFTkSuQmCC" />
            </td>
        </tr>
    </table>
 </div>